erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json

```diff
@@ -40,6 +41,7 @@
   "base_amount",
   "pricing_rules",
   "is_free_item",
```

Comment:
move the checkbox to the bottom of the column. Alignment is messy as it is at the top